src/helm/benchmark/augmentations/cleva_perturbation.py

```diff
@@ -43,7 +43,15 @@ class Description(PerturbationDescription):
     name: str = "chinese_typos"
 
     # For downloading resources
-    ASSET_URL = "https://drive.google.com/uc?id=1p5mldLpKxI-63H8YEruGJghtD1dZJI8k"
+    ASSET_URL = "http://39.108.215.175/assets/butter_finger"
```

Comment:
In general, it is better to have this be a domain name due to (1) HTTPS security and (2) it's easier to lose an allocated IP address than a domain name. Is there any plan to change the URL back to a HTTPS domain name?

For instance, these assets could be hosted in a GitHub repository.

Reply:
Because of the [Great Firewall](https://en.wikipedia.org/wiki/Great_Firewall), Google Drive is not accessible in China. For this reason, and to ensure our project's long-term stability, we have rented a cloud server with a two-year contract. This means that this IP address and related service will remain stable in the foreseeable future.

In the meantime, we are applying for a domain and an SSL certificate. We estimate the process should be completed within 2 weeks. Immediately after, we will update the URLs to use the new HTTPS domain name.

Please let me know how you feel about this plan. We truly value your suggestions. Thank you!